communitiesuk/funding-service-design-fund-store
# Fs 4035 cof eoi schema

### Change description
- Created EOI schema for COF
- Added this to the round loader for EOI
- Updated all other rounds to have a `None` value for that field
- Updated test data accordingly
- Excluded the `eoi_decision_schema` from the `Round` schema so it's not returned in existing APOI calls
- Added a new endpoint to return that schema



### How to test
Pull this branch
Update the DB with the latest round data
Run this branch and then navigate to this URL to see the cof eoi schema
http://127.0.0.1:5000/funds/cof-eoi/rounds/r1/eoi_decision_schema?use_short_name=true


### Screenshots of UI changes (if applicable)
